invenio_jobs/services/schema.py

```diff
@@ -105,6 +114,31 @@ def get_obj_type(self, obj):
     type_field_remove = False
 
 
+class RegisteredTaskArgumentsSchema(OneOfSchema):
+    """Base schema for tasks with arguments."""
+
+    type_field_remove = False
+
+    def __init__(self, *args, **kwargs):
+        """Constructor."""
+        self.type_schemas = deepcopy(current_jobs.registry.registered_schemas())
```

Comment:
minor: can this be "lazy", via e.g. a property? In general when we start using `current_xxx` and Flask app-context in `__init__`, we end up in difficult situations in config/schemas.